### DISCUSSION TITLE
Add grass or bush for the player, so they can hide from the enemy :thonk:

### DISCUSSION THREAD

**Discussion by Daria13szamman** [Quit975/DuckyGame#50] [Ideas]
Bush or grass placed randomly/intentionally on the screen so the player can hide in them. We can do limited time for hiding, maybe :thonk:

**Comment by Quit975** [comment]
These special places could also depend on the biome we play on if we would follow Natalia's idea https://github.com/Quit975/DuckyGame/discussions/55
